frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -54,11 +57,24 @@ export default function ModelAccess({ model }: ModelAccessProps) {
 
   // TODO - add a request to update the model's collaborators field
   async function updateAccessList() {
-    await patchModel(model.id, { collaborators: accessList })
-  }
+    setLoading(true)
+    const response = await patchModel(model.id, { collaborators: accessList })
+
+    if (!response.ok) {
+      const error = await getErrorMessage(response)
+      setLoading(false)
+      setPatchModelErrorMessage(error)
+    }
+
+    // TODO - Provide user feedback on success
 
-  if (isUsersError) {
-    return <MessageAlert message={isUsersError.info.message} severity='error' />
+    if (patchModelErrorMessage) {
```

Comment:
The following should not be inside this function:
```
    if (patchModelErrorMessage) {
      return <MessageAlert message={patchModelErrorMessage} severity='error' />
    }

    if (isUsersError) {
      return <MessageAlert message={isUsersError.info.message} severity='error' />
    }
```

They are used to return a `MessageAlert` instead of the main `return` block at the bottom in the event of an error occurring. If they are inside this function they will return the `MessageAlert` to whatever called `updateAccessList`